### DISCUSSION TITLE
[Feature] Small quick-win UI changes, while editing faces

### DISCUSSION THREAD

**Discussion by brayStorm** [immich-app/immich#12657] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

My model seems to think my wife and my sister in-law are the same person. 🙃

### Web
1. I'm on the `[immich_url]/people/[person_id]/photos/[photo_id]` page. **In the info dialog, it would be nice to have a toggle to show/hide marked faces, and have it default to the [person_id] I'm navigated to.** 
Right now I have to mouse-over each time to see if my sister-in law is in the photo or not (most are wedding group photos).2

2. Then when I return to `[immich_url]/people/[person_id]?at=[unknown_id]`, it jumps all the way back to the top. **It should return me to that photo**, so I can continue on.

3. When editing faces from this dialog, I'm forced to select another recognized face, even if I don't know who it is. **Add option to label unknown at the top of the list.**

4. When there are no recognized faces in a photo, **add the ability to manually mark one in the info dialog.**

### Mobile
5. In the mobile app, when trying to scroll to the bottom of the Photos timeline, the scroll bar maxes out below the footer. **Change the limits on the length of the scroll bar and make the bottom 15% of it continuously scroll.**

6. **Add the ability to edit faces from the mobile app.**

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
These are several different feature requests, a few of which are duplicates.
